cloudbuild.yaml

```
@@ -1,12 +1,16 @@
 steps:
   - id: 'build'
-    name: 'earthly/earthly:v0.5.24'
+    name: 'earthly/earthly:v0.7.23'
```

Use the new one that just came out?

```suggestion
    name: 'earthly/earthly:v0.8.0'
```

---

It released between when I started and when I put this up. I can do that